src/ventures-for-shared-prosperity/index.njk

```diff
@@ -7,63 +7,59 @@ title: Ventures for Shared Prosperity
 {% block masthead %}
   {{ super() }}
   <div class="hero hero--centered standard-restricted-width">
+  <img src="" alt="colorful octogon"/>
+  <p>cohort one: 2021-2022</p>
     <h1 class="hero__title">
       Ventures for Shared Prosperity
     </h1>
+    <p> Improving the financial health and well-being of low and moderate income people in America.</p>
+
+    {%include "components/sub-banner.njk" %}
+
+    <div>
```

Comment:
Exact location TBD, but we want to move some of this content. Right now, every you have here is in the `masthead` block. That's not what we want. On the page: https://deploy-preview-29--ideas42ventures.netlify.app/ventures-for-shared-prosperity/  scroll all the way to the bottom. Notice how there's a gap of white space below the purple footer? That's because the entire page is in `masthead` that has padding applied to it.

Reply:
Fixed this!

Reply:
Fixed!